packages/trader/src/App/Containers/trade-footer-extensions.jsx

```diff
@@ -34,6 +34,8 @@ const TradeFooterExtensions = ({
                     Component: TogglePositionsIcon,
                 },
             ]);
+        } else {
+            populateFooterExtensions([]);
```

Comment:
I think with this change the effect on line 17 can be removed?

Reply:
> I think with this change the effect on line 17 can be removed?

No. it is needed too. e.g. when a user goes from trader to cashier, without that line, the icon stays in the footer.